services/spar/test-integration/Test/Spar/APISpec.hs

```diff
@@ -228,6 +235,7 @@ spec = do
 
       context "some URLs are not https" $ do
```

Comment:
Not sure if you're aware of the `HttpsUrl` type defined [here](https://github.com/wireapp/wire-server/blob/develop/libs/types-common/src/Data/Misc.hs#L194-L227) and if that could be useful.